students/student_framllat/session02/series.py

```diff
@@ -0,0 +1,69 @@
+a = int(input("How many nums shall I calculate for you? "))
+
+def fibonacci(n):
+    
+    """This function calculates the nth value of a Fibonacci number
+     starting at the zero index
+    """
+
+    if n == 0:
+        return 0
+    elif n == 1:
+        return 1
+    else:
+        return fibonacci(n - 2) + fibonacci(n - 1)
+
+def lucas(n):
+
+    """This function calculates the nth value of a Lucas Number
+     starting at the zero index
+    """
+
+    if n == 0:
+        return 2
+    elif n == 1:
+        return 1
+    else:
+        return lucas(n - 2) + lucas(n - 1)
+
+def sum_series(n, n0=0, n1=1):
+
+    """This function attempts to calculate the nth value of a Series 
+     starting at the zero index
+    """
+
+    if n == 0:
+        return n0
+    elif n == 1:
+        return n1
+    else:
+        return sum_series(n - 2) + sum_series(n - 1)
```

Comment:
and here we have our problem -- when you call this will n > 1, it sill recursively call itself with smaller numbers until it terminates at the "bottom" -- n = 0 or 1. But when you calll it this way -- it has the samller n, but how does it know what n0 and n1 to use?